index.html

```diff
@@ -44,6 +44,194 @@ <h1>Hey There. I'm Bright</h1>
+      </p>
+      <p class="profesional-Art-content">
+        A daily
+        selection of privately personalized reads;
+        no accounts or sign-ups required. has been the
+        industry's standard
+      </p>
+      <ul class="profesional-Art-ul">
+        <li class="profesional-Art-li">html</li>
+        <li class="profesional-Art-li">bootstrap</li>
+        <li class="profesional-Art-li">Ruby</li>
+      </ul>
+      <a href="#4" class="profesional-Art-btm button">See Project</a>
+    </div>
+    <div class="art-container">
+      <p class="profesional-Art-heading">
+        Profesional Art Printing Data
+      </p>
+      <p class="profesional-Art-content">
+        A daily
+        selection of privately personalized reads;
+        no accounts or sign-ups required. has been the
+        industry's standard
+      </p>
+      <ul class="profesional-Art-ul">
+        <li class="profesional-Art-li">html</li>
+        <li class="profesional-Art-li">bootstrap</li>
+        <li class="profesional-Art-li">Ruby</li>
+      </ul>
+      <a href="#5" class="profesional-Art-btm button">See Project</a>
+    </div>
+    <div class="art-container">
+      <p class="profesional-Art-heading">
+        Profesional Art Printing Data
+      </p>
+      <p class="profesional-Art-content">
+        A daily
+        selection of privately personalized reads;
+        no accounts or sign-ups required. has been the
+        industry's standard
+      </p>
+      <ul class="profesional-Art-ul">
+        <li class="profesional-Art-li">html</li>
+        <li class="profesional-Art-li">bootstrap</li>
+        <li class="profesional-Art-li">Ruby</li>
+      </ul>
+      <a href="#6" class="profesional-Art-btm button">See Project</a>
+    </div>
+    <div class="art-container">
+      <p class="profesional-Art-heading">
+        Profesional Art Printing Data
+      </p>
+      <p class="profesional-Art-content">
+        A daily
+        selection of privately personalized reads;
+        no accounts or sign-ups required. has been the
+        industry's standard
+      </p>
+      <ul class="profesional-Art-ul">
+        <li class="profesional-Art-li">html</li>
+        <li class="profesional-Art-li">bootstrap</li>
+        <li class="profesional-Art-li">Ruby</li>
+      </ul>
+      <a href="#7" class="profesional-Art-btm button">See Project</a>
+    </div>
+  </section>
+  <section class="about">
+    <div class="about-me-footer">
+      <div>
+        <h2>About me</h2>
+        <p class="footer-text">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience!
+          If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <a href="#8" class="footer-btm button">Get My Resume</a>
+      </div>
+      <div class="footer-img">
+        <img src="/Images/illustration-about-me-mobile.png" alt="">
```

Comment:
You have done a fantastic job so far😉😉👍 Just this issue🥲 Kindly ensure an alt attribute is always added, and it shouldn't be left empty😉 this makes your project accessible to screen readers and it is also a best practice in html🤗🤗 Kindly make the necessary change🤗🤗